components/checkbox/overview.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Overview
+page_title: Checkbox Component for Blazor Overview
+description: Overview of the Checkbox for Blazor
+slug: components-checkbox-overview
+tags: telerik,blazor,checkbox,overview
+published: True
+position: 0
+---
+
+# Checkbox Overview
+
+The Checkbox component allows you to add more customizable checkboxes to your Blazor application. It maintains the behavior of the standard HTML checkbox and provides checked, unchecked and indeterminate states.
+
+To use a Telerik Checkbox for Blazor
+
+1. add the `TelerikCheckBox` tag
+1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. (optional) set the `Indeterminate` or `bind-Indeterminate` property
```

Comment:
This should not be here, there should be a separate "Indeterminate State" article that explains what it is and how to use it. The first example should be short, concise and not to have many "optional" items.

Reply:
The example below is great, but the article should start with a much simpler one that shows a comparison between a "regular" checkbox and an indetermined one to showcase the difference. Then, add the second example immediately after. It is pretty actionable, so you could add a gif of it as well.